src/index.js

```diff
@@ -7,43 +7,61 @@ import transformCall from './transformers/call';
 import transformImport from './transformers/import';
 
 
-const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
-export const defaultExtensions = defaultBabelExtensions;
```

Comment:
Exporting them was required for the eslint plugin, instead of duplicating these values in both projects. (https://github.com/tleunen/eslint-import-resolver-babel-module)
https://github.com/tleunen/eslint-import-resolver-babel-module/blob/master/src/index.js#L66-L68